### DISCUSSION TITLE
Champs manquants dans le XML sur eNotices

### DISCUSSION THREAD

**Discussion by ismailatitallah** [OP-TED/eForms-SDK#717] [Q&A] [answered]
Bonjour, 

Ci-joint un zip qui contient un xml envoyé à TED. Et un xml téléchargé depuis le TED après la soumission.

Lorsqu’on vérifie le contenu de la notice XML d’un avis dans eNotices, on se rend compte que plusieurs champs ne sont pas repris.

Liste non exhaustive : 

- cbc:ContractingSystemTypeCode listName="dps-usage"

- cbc:CriterionTypeCode listName="selection-criterion" (un seul critère de sélection est repris)

- efac:Organization (une seule organisation est reprise, un mélange a lieu avec le numéro de fax)

on utilise la version 1.5.4. Est ce que cette anomalie est corrigée? 

[xml.zip](https://github.com/OP-TED/eForms-SDK/files/12883464/xml.zip)

**Comment by YvesJo** [comment, marked as answer]
Please get in touch directly with the helpdesk ted@publications.europa.eu
